attributes/default.rb

```diff
@@ -18,6 +18,7 @@
 #
 # TODO: (jtimberman) Configure this from a Delivery `config.json`.
 default['habitat-build']['depot-url'] = 'https://willem.habitat.sh'
+default['habitat-build']['hab-version'] = '0.33.2'
```

Comment:
It’s ok to expose this as an attribute but it would theoretically be undefined behaviour if anyone bumped the version up or down in this manner. You might want to consider hard coding this and releasing a new cookbook each time you change it